modules/ws/src/test/resources/ws-consumer-url-connection-test-case.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
+	xmlns:ws="http://www.mulesoft.org/schema/mule/ws"
+	xsi:schemaLocation="
+        http://www.mulesoft.org/schema/mule/ws http://www.mulesoft.org/schema/mule/ws/current/mule-ws.xsd        
+        http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+        http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+        http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd">
+
+	<http:request-config name="HTTP_Request_Configuration"
+		protocol="HTTPS" host="fake.com" responseTimeout="60000" port="80"
```

Comment:
This is never called. Only the initialization is performed as the wsdl retrieval is tested (it is retrieved either from a local file or from a server in localhost via http). However, I removed the timeout attribute.